### PR TITLE
[v2.0.3]: Fixed issue #3124

### PR DESCRIPTION
**Pull request by Kantharajut-btc**
Fixed issue #3124: [Enroll Module]

- Migrated the Spring Boot version to 2.2.6.RELEASE for consistency
- Removed Spring Security dependency and it's configuration as we are using Hydra for resource access
- Removed log4j.xml and configured logging in application.properties for consistency

### CODE REVIEW

File: participant-datastore/enroll-mgmt-module/enroll-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/UserEnrollmentTokenMgmtServiceWsApplication.java [hunk 1]
```diff
@@ -13,7 +13,9 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.domain.EntityScan;
 import org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration;
+import org.springframework.scheduling.annotation.EnableScheduling;
```
Comment (moschetti) [inline participant-datastore/enroll-mgmt-module/enroll-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/UserEnrollmentTokenMgmtServiceWsApplication.java:16]: Please update file copyright to `2020-2021`

File: participant-datastore/enroll-mgmt-module/enroll-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/config/AppConfig.java [hunk 2]
```diff
@@ -8,7 +8,15 @@
 
 package com.google.cloud.healthcare.fdamystudies.config;
 
+import org.springframework.context.annotation.Bean;
```
Comment (moschetti) [inline participant-datastore/enroll-mgmt-module/enroll-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/config/AppConfig.java:11]: Please update file copyright to `2020-2021`

File: participant-datastore/enroll-mgmt-module/enroll-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/UserEnrollmentTokenMgmtServiceWsApplication.java (hunk 1 above)
Comment (Kantharajut-btc) [reply participant-datastore/enroll-mgmt-module/enroll-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/UserEnrollmentTokenMgmtServiceWsApplication.java:16]: Modified copyright text

File: participant-datastore/enroll-mgmt-module/enroll-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/config/AppConfig.java (hunk 2 above)
Comment (Kantharajut-btc) [reply participant-datastore/enroll-mgmt-module/enroll-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/config/AppConfig.java:11]: Modified copyright text

**Comment by Kantharajut-btc** [comment]
@moschetti  please review and approve PR

**Review by moschetti — APPROVED** [review]

**Pull request merged**